test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -671,6 +672,15 @@ def __get_param_dictionary(self):
         Method to convert data in param into dictionary of cluster
         information
         """
+        def get_bool(v):
+            if v == None or v == '':
+                return False
```

Comment:
Shouldn't you say:
```
if not v:
```

Reply:
I could have done that, but I wanted to be explicit about what I was testing for.